### ISSUE TITLE
Update the container of an equipement is impossible

### ISSUE THREAD

**Issue by EmmaDorie** [ekylibre/ekylibre#751]
When an equipment is already used in interventions, the field to select the container disappear.
![screenshot - 16062016 - 18 44 26](https://cloud.githubusercontent.com/assets/11627573/16125136/8cfb33ea-33f2-11e6-9b1d-760722d4db49.png)

**Comment by Nyxelio** [comment]
Fields disappearing isn't linked interventions but page caching. Manually refresh the page will do the job.

**Comment by Nicephore17** [comment]
Manual refresh don't solve the issue for me :(

**Comment by PH64** [comment]
Are you sure ? it seems it don't work... The problem is the same with matter

**Comment by Nicephore17** [comment]
Yes, when I try to edit a Equipement used in an intervention, I loss eves fields after **variety**.
No more access to: **Initial owner**, **container**, **geolocation** nor **born** an **death** dates.
Refresh page (Firefox on Windows 10) change nothing. Anything to do with the "done" status (see #738 )?

**Comment by PH64** [comment]
@Nicephore17 I agree with you. This is with @Nyxelio that I'm not agree.
"done status" is from Ekylibre V1 (in which we could do forecast intervention) and I suppose it will be used in V3 with planification module

**Comment by Nicephore17** [comment]
OK ! Could you update #738 ? I wonder if you will expect a production/order forecast <3 !

**Comment by Nyxelio** [comment]
Oh no, it'll not work for you as you need to make a server-side update. I'm investigating.

**Comment by Nyxelio** [comment]
What is the purpose of changing the localization of the equipment ?

**Comment by Nicephore17** [comment]
Geolocation ? No specific use, I was just listing the differences. Actually, if the equipment as a geolocation system that works with Ekylibre, you should keep de the map no ?

I was more concerned by the possibility to correct born date if needed and to update death date or change the container !!

**Comment by PH64** [comment]
The main goal is to have a proper inventory, the inventory is done by storage place.
If a mistake is made in the creation of the material, you can not change

**Comment by Nicephore17** [comment]
I understand but the behavior is inconsistent then.
Some suggestions...

**Option 1**
Equipment is available in inventory and you can setup the container when creating. Then it should be possible to update container as long as you can decide ton change it. Ex: new equipment, new parking zone etc.

**Option 2**
Equipment is not listed as stock and you can change the container as you want to track the appropriate parking area.

**Option 3**
Equipment is automatically stored in a virtual container (_Equipment list_ ?) so it allows to make an inventory of the equipment wherever it could be actually stored. Then, this virtual container should not show up on the building map. Good point: no possible mistake when adding new equipment to the farm fleet.

**Comment by PH64** [comment]
@Nicephore17, I agree with you. I explained to @Nyxelio why store place must be change.

**Comment by Nicephore17** [comment]
LOL, sorry to much heat today, my poor brain....

**Comment by PH64** [comment]
@Nyxelio , the problem is exactly the same with Matters

**Comment by Nyxelio** [comment]
Ok I get it, I will add fields to create/update localizations. A localization defines the product container for a given time.

**Comment by PH64** [comment]
@Nyxelio , to summarize, Modification screen is different from Creation

Creation :
![capture du 2016-06-22 20-11-41](https://cloud.githubusercontent.com/assets/12525665/16278213/554bca68-38b6-11e6-9696-2568d15bb987.png)

Modification : 
![capture du 2016-06-22 20-13-01](https://cloud.githubusercontent.com/assets/12525665/16278219/5f1fcdd2-38b6-11e6-8e12-d9e55d34d6c6.png)

**Comment by Nyxelio** [comment]
Sure, I agree with both of you, edit form is different of creation form. But it's actually the expected behavior as we consider we can't change life dates (if and only if the product is already used in interventions).

Moreover, I agree to say form display lacks consistency, as all fields was sometimes displayed even if it is already in intervention. That's fixed.

Next, I add possibility to add many localizations for a product during its life.

To summarize, I will add possibility to edit dates (and check intervention consistency).
I hope that's clearer.
